CHANGELOG.md

```diff
@@ -1,4 +1,80 @@
 # Changelog
+## Version 3.7.0 - Mar 6, 2023
+
+Version 3.7.0 has bug fixes related to m2studio variable cards, updating to the latest slate editor, restricting to one diagram tile per document, shared models in curriculum sections among other features.
+
+### Features/Improvements
+- Restrict to One Diagram Tile [#184008021](https://www.pivotaltracker.com/story/show/184008021)
+- Insert Variable on Text Toolbar [#183806331](https://www.pivotaltracker.com/story/show/183806331)
+- New/Edit Variable from Text Toolbar [#183534556](https://www.pivotaltracker.com/story/show/183534556)
+- Update slate-editor library to latest slate version [#181890362](https://www.pivotaltracker.com/story/show/181890362)
+- Geometry Tile should preserve connected shapes [#183053771](https://www.pivotaltracker.com/story/show/183053771)
+- Link Table/Geometry Axis Domain Synchronization [#184187800](https://www.pivotaltracker.com/story/show/184187800)
+- Warning on mismatched units [#184365058](https://www.pivotaltracker.com/story/show/184365058)
+- Restyle warnings [#184316798](https://www.pivotaltracker.com/story/show/184316798)
+- Warning on unknown variables [#184272677](https://www.pivotaltracker.com/story/show/184272677)
+- Edit Variable dialog  displays read only expressions when used [#183974159](https://www.pivotaltracker.com/story/show/183974159)
+- Warning on missing operands [#184258400](https://www.pivotaltracker.com/story/show/184258400)
+- Selected text should NOT be replaced by variable chip. Instead, selected text should stay in the text tile and populate the Notes field of the variable.  [#184395017](https://www.pivotaltracker.com/story/show/184395017)
+- Hyperlinks operation in write mode (**text tile**) [#182820037](https://www.pivotaltracker.com/story/show/182820037)
+- <H> Log Tile Titles [#184428587](https://www.pivotaltracker.com/story/show/184428587)
```

Comment:
Not a big deal, but is there a reason for these `<H>` tags that occur here and a few other places?

Reply:
For some reason, they are in the story titles.